packages/tree-widget/src/components/trees/ClassificationsTree.tsx

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Copyright needs to be updated with OSS one. Please copy from other files in the repo